python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -325,25 +325,20 @@ def printSummaryTree(self, level=0, verbosity_level=0, whitespace_marker='', out
         from Ganga.GPIDev.Base.VPrinter import VSummaryPrinter
         self.accept(VSummaryPrinter(level, verbosity_level, whitespace_marker, out, selection, interactive))
 
-    def __eq__(self, _node):
-
-        node = _node
-
+    def __eq__(self, node):
         if self is node:
-            return 1
-        if not node:  # or not self._schema.isEqual(node._schema):
-            return 0
+            return True
```

Comment:
This could cause problems with the Schema which makes a lot of use of ```0``` and ```1``` rather than ```True``` and ```False```